docs/en/observability/logs-index-template.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[logs-index-template]]
+= Logs index template reference
+
+[[custom-logs-index-template]]
+== View index templates
+
+Index templates are used to configure the backing indices of data streams as they are created.
+These index templates are composed of multiple component templates--reusable building blocks
+that configure index mappings, settings, and aliases.
+
+The default Logs index template can be viewed in {kib}.
+Navigate to **{stack-manage-app}** → **Index Management** → **Index Templates**, and search for `logs`.
+Select any of the logs index templates to view their relevant component templates.
+
+[discrete]
+[custom-logs-template-edit]
+=== Edit the {es} index template
+
+{es} has a built-in index template for the `logs-*-*` index pattern, which is composed of component templates including:
+
+* `logs@mappings`
+* `logs@settings`
+* `logs@custom`
+* `ecs@mappings`
+
+You can use the `@custom` component template to customize your {es} indices. The `logs@custom` template is not installed by default, but you can create a template named `logs@custom` to override and extend default mappings or settings.
```

Comment:
I currently get this error because logs@custom doesn't exist. I'm using this version of the stack `8.13.0-f56d9bd5-SNAPSHOT` released on 2024-02-05

```json
{
  "statusCode": 400,
  "error": "Bad Request",
  "message": "index_template [simulate_template_cjga1q24s0swusgjolwsba] invalid, cause [index template [simulate_template_cjga1q24s0swusgjolwsba] specifies component templates [logs@custom] that do not exist]",
  "attributes": {
    "error": {
      "root_cause": [
        {
          "type": "invalid_index_template_exception",
          "reason": "index_template [simulate_template_cjga1q24s0swusgjolwsba] invalid, cause [index template [simulate_template_cjga1q24s0swusgjolwsba] specifies component templates [logs@custom] that do not exist]"
        }
      ],
      "type": "invalid_index_template_exception",
      "reason": "index_template [simulate_template_cjga1q24s0swusgjolwsba] invalid, cause [index template [simulate_template_cjga1q24s0swusgjolwsba] specifies component templates [logs@custom] that do not exist]"
    }
  }
}
```

Reply:
I've encountered this at some point (see [thread](https://elastic.slack.com/archives/CNMNXV4RG/p1707391367379259
)) but couldn't replicate it today after testing several versions (didn't test snapshots)

Reply:
@gbamparop I could use someone else trying the latest snapshot to see if it is just me or we need to open a bug about this.

Reply:
@gbamparop I have added some more info to this [thread](https://elastic.slack.com/archives/CNMNXV4RG/p1708015894378669?thread_ts=1707391367.379259&cid=CNMNXV4RG) so that someone else can replicate it.